core/controllers/contributor_dashboard.py

```diff
@@ -361,54 +360,31 @@ def _get_reviewable_exploration_opportunity_summaries(
         Returns:
             list(ExplorationOpportunitySummary). A list of the matching
             exploration opportunity summaries.
-
-        Raises:
-            Exception. No exploration_id found for the node_id.
         """
-        # 1. Fetch the eligible topics.
-        # 2. Fetch the stories for the topics.
-        # 3. Get the reviewable translation suggestion target IDs for the user.
-        # 4. Get story exploration nodes in order, filtering for explorations
-        # that have in review translation suggestions.
-        if topic_name is None:
-            topics = topic_fetchers.get_all_topics()
-        else:
+        # 1. Fetch all the exploration IDs of chapters in stories under the
+        #    topic(s).
+        # 2. Get the reviewable translation suggestion target IDs for the user.
+        # 3. Get story exploration nodes in order, filtering for explorations
+        #    that have in review translation suggestions.
+        # 4. Fetch all exploration opportunity summaries, order them, and then
+        #    add the pinned lesson's summary on top.
+        if topic_name:
             topic = topic_fetchers.get_topic_by_name(topic_name)
             if topic is None:
                 raise self.InvalidInputException(
                     'The supplied input topic: %s is not valid' % topic_name)
-            topics = [topic]
-        topic_stories = story_fetchers.get_stories_by_ids(
-            [
-                reference.story_id
-                for topic in topics
-                for reference in topic.get_all_story_references()
-                if reference.story_is_published
-            ],
-            strict=False
-        )
-        topic_exp_ids = []
-        for story in topic_stories:
-            if story is None:
-                continue
-            for node in story.story_contents.get_ordered_nodes():
-                if node.exploration_id is None:
-                    raise Exception(
-                        'No exploration_id found for the node_id: %s'
-                        % node.id
-                    )
-                topic_exp_ids.append(node.exploration_id)
+            topic_exp_ids = (
+                topic_fetchers.get_published_story_exploration_ids(topic.id))
```

Comment:
Note: for params with default args, please include the argname before the argument when calling them.

Reply:
Done.